Profile Folder/chrome/styles/partials/content/chrome-1/controls/_native-overrides.scss

```diff
@@ -0,0 +1,12 @@
+@-moz-document	url-prefix(about:preferences),
```

Comment:
I'd say it's better to remove this for now. I can't seem to replicate the dropdown icon fix in every browser (it just refuses to work on Linux Firefox 132 for me) and I don't even remember how the custom dropdown icon is even set.

Reply:
With this we can go with two solutions:
1. Add a condition to restrict it to Windows.
2. Hold back that fix as you suggested it.

I'm OK with the two solutions. As the maintainer, I let you decide what you want to do with this half-baked fix (after all I was aware that this "fix" was kinda wobbly since minute one).

Reply:
I prefer to hold back if you don't mind, I don't really like the dropdown stylecode in the first place :/

Reply:
Deal. I'll revert that one, the other with the text color will stay.

Reply:
Done.
[`eb338f4`](https://github.com/angelbruni/Geckium/pull/328/commits/eb338f40556b99058443e33564bf3026b2937dfa)